rfd/0047-webui-paginate-filter.md

```diff
@@ -0,0 +1,233 @@
+---
+authors: Lisa Kim (lisa@goteleport.com)
+state: draft
+---
+
+# RFD 47 - WebUI server-side paginating, searching, and filtering for servers.
```

Comment:
Note: I think we have 3 open PRs that have all claimed RFD #47. Maybe we need to keep a separate list somewhere so you can grab a number before you start and ensure that other RFDs don't conflict.

Reply:
ah thanks for pointing that out, what i will do is instead of rfd `number`, i'll write `tbd`, and before I merge replace `tbd` with the next number from master, is this okay? Btw I created a new PR b/c my branch also had the number